src/common/CommonUtils.ts

```diff
@@ -29,6 +29,11 @@ export class CommonUtils {
         promise: Promise<T>,
         failureMessage?: string
     ): Promise<T> {
+        // Javascript/TypeScript do not have promises that timeout. However we could use
```

Comment:
Oh sorry I just meant the JSDoc. The method made sense to me, but given our eventual mission to JSDoc everything, I was suggesting we make sure we cover it here too.